.travis.yml

```diff
@@ -50,7 +50,6 @@ matrix:
                       - binutils-mingw-w64-i686 gcc-mingw-w64-i686 g++-mingw-w64-i686
 
 script:
-    - set -o pipefail
-    - git diff `git merge-base master HEAD` | ./tools/scripts/checkpatch.pl -
+    - set -o pipefail && git diff `git merge-base master HEAD` | ./tools/scripts/checkpatch.pl -
```

Comment:
FWIW when I run this locally I use `riscv` instead of `master`. Not sure if that would help you fail automatically? Or maybe you should make this its own test step?